build-scripts/create-nightly-version.js

```diff
@@ -33,29 +31,11 @@ const nightlyVersion = `${today.getFullYear()}${month}${day}.0.0-nightly`;
 
 (async () => {
   try {
-    // Lerna won't release packages on an already release tagged commit or
-    // on a disconnected HEAD. Create a branch then commit the os changes for this nightly release.
+    // Create a branch then commit the changes for this nightly release.
     await runCommand('git', ['checkout', '-b', `publish-${nightlyVersion}`]);
     await runCommand('git', ['add', 'compiler']);
-    await runCommand('git', ['add', 'packages/google-closure-compiler-linux/package.json']);
-    await runCommand('git', ['add', 'packages/google-closure-compiler-osx/package.json']);
-    await runCommand('git', ['add', 'packages/google-closure-compiler-windows/package.json']);
-    await runCommand('git', ['commit', '-m', `Create version for nightly release ${nightlyVersion}`]);
-    // Get the list of packages in this repo
-    const packages = glob.sync('packages/google-closure-compiler*')
-        .map(packagePath => packagePath.replace('packages/', ''));
-
-    // Create a nightly version of all the packages
-    await runCommand(
-      'node',
-      [
-        './build-scripts/lerna-publish.js',
-        'version',
-        nightlyVersion,
-        '--push=false', // prevent the version commit from being pushed back to the repo
-        `--force-publish=${packages.join(',')}`, // publish every package even though no changes are detected
-        '--yes', // don't prompt for confirmation
-      ]);
+    await runCommand('yarn', ['config', 'set', 'version-git-message', `Create version for nightly release ${nightlyVersion}`]);
```

Comment:
would it be cleaner to use `yarn version --new-version [nightly] --message "Create version %s"`, instead of setting a permanent config flag?

Reply:
I'd love to do this - but I can't find any documentation for a `--message` flag in Yarn 1.x. Do  you have a link?

Reply:
That's amazing. Thanks. I specifically went to their public web docs looking for this capability and was very annoyed it wasn't there.